Tools/Scripts/libraries/webkitcorepy/webkitcorepy/task_pool.py

```diff
@@ -407,12 +409,13 @@ def do(self, function, *args, **kwargs):
 
         if callback:
             self.callbacks[self._id_count] = callback
+        self.pending += 1
```

Comment:
nit: I'd call it pending_count or num_pending.  Something that indicates a number as opposed to a state